themes/mCommerceTheme/styles/components/LviAddReviewSection.json

```diff
@@ -0,0 +1,7 @@
+{
+	".lviAddReviewSection": {
+		"height": 70,
+		"backgroundColor": "${backgroundMain}",
```

Comment:
remove backgroundColor if background on the page is same as this

Reply:
It is necessary because of ios.